pkg/controller/secret/secretsync.go

```diff
@@ -161,13 +168,23 @@ func (c *SecretSyncController) translateSecret(cloudSecret *v1.Secret, cloudConf
 	}
 	accountID := match[1]
 
+	iamEndpoint, err := fetchEndpoint("iamEndpointOverride", conf, defaultTokenExchangeURL)
+	if err != nil {
+		return nil, err
+	}
+
+	riaasEndpoint, err := fetchEndpoint("g2EndpointOverride", conf, fmt.Sprintf(defaultRIAASEndpointURL, region))
+	if err != nil {
+		return nil, err
+	}
+
 	resourceId, err := c.getResourceID(resourceGroupName, accountID, string(apiKey))
```

Comment:
This call is using the default IAM endpoint, and a hardcoded ResourceManager endpoint.
Both should be able to be overridden, which means you will also want to get the ResourceManager endpoint from the `cloud-conf` too.

At least best I can tell, the defaultGetResourceID function is being called, which uses these default endpoints
https://github.com/openshift/ibm-vpc-block-csi-driver-operator/blob/master/pkg/controller/secret/secretsync.go#L185-L189